src/frontend/citizen-portal/src/app/services/notice-of-dispute.service.ts

```diff
@@ -164,6 +164,12 @@ export class NoticeOfDisputeService {
 
     input.dispute_counts.forEach(count => { // TODO: remove this once request_court_appearance removed from dispute count schema, API
```

Comment:
`input.dispute_counts.forEach((count: any) => {`
Or there should have a class have `__skip`, you can replace `any` with that class

Reply:
Same as above. The declared class doesn't have this member.

Reply:
Yeah, sorry, this doesn't work. If the input parameter is strongly typed, I get this:
![image](https://github.com/bcgov/jag-traffic-courts-online/assets/55215368/1ab49cf5-e8e6-4c14-81dd-00bf6eadba47)

Yet, as you mentioned, the __skip is in fact on the DisputeCount object. 
![image](https://github.com/bcgov/jag-traffic-courts-online/assets/55215368/053efec2-2a3c-4b70-b6a9-eae2c2b36a96)

I agree, the method signature should be strongly type, but the compiler refuses to recognize the property. Not sure how to fix this.

Reply:
Yeah, I love the strongly typed TypeScript in the citizen-portal, so thanks for that! The code is much easier to understand than the staff-portal.